docs/cloud/satellites/managing.md

```diff
@@ -98,22 +98,87 @@ Currently selected: No
 
 ### Clearing cache
 
-To clear the cache of a satellite, run the following while a satellite is selected:
+The easiest way to clear cache on a satellite is to restart it using the `update` command with `--drop-cache` flag.
```

Comment:
I would rephrase this to mention there's two different ways (rather than easier vs harder)

maybe something along the lines of:

```
There are two ways to clear the satellite cache:

#### recreating the underlying satellite instance (often faster)

    ```bash
    earthly satellite update --drop-cache my-satellite
    ```

Note that this operation can take a while, and the satellite may also receive any available updates during the process.

#### Using the prune command (slower)

The `earthly prune` command can also be used. In many cases, however, it takes longer than using `satellite update`.
The prune command also requires the satellite to be selected before running.

    ```bash
    earthly prune -a
    ```

```

Reply:
I like it, thanks!